pkg/server/nfs/export.go

```diff
@@ -154,6 +159,53 @@ func (e *exporter) DeleteExport(volume string) error {
 	return nil
 }
 
+func (e *Exporter) ReloadExport() error {
+	processName := "ganesha.nfsd"
+
+	pid := findPidByName(processName)
+	if pid == -1 {
+		return fmt.Errorf("process %s is not found", processName)
+	}
+
+	process, err := os.FindProcess(pid)
+	if err != nil {
+		return fmt.Errorf("failed to find process %s", processName)
+	}
+
+	err = process.Signal(syscall.SIGHUP)
+	if err != nil {
+		return fmt.Errorf("failed to send SIGHUP to process %s", processName)
+	}
+
+	return nil
+}
+
+func findPidByName(name string) int {
```

Comment:
Suggest doing the same as `os.FindProcess` to return error instead of using num as return code.

Also, reducing system CLI usage will help decouple from the host dependency. Probably can use https://github.com/mitchellh/go-ps/blob/master/process_unix.go#L50 this lib instead.

Reply:
Addressed. Thank you.